lib/src/test/java/space/taran/arklib/user/tags/AggregateTagStorageTest.kt

```diff
@@ -0,0 +1,139 @@
+package space.taran.arklib.user.tags
+
+import dev.arkbuilders.arklib.ResourceId
+import dev.arkbuilders.arklib.data.index.RootIndex
+import dev.arkbuilders.arklib.user.tags.AggregateTagStorage
+import dev.arkbuilders.arklib.user.tags.RootTagsStorage
+import dev.arkbuilders.arklib.user.tags.Tag
+import dev.arkbuilders.arklib.user.tags.Tags
+import io.mockk.CapturingSlot
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.unmockkAll
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import java.util.*
+
+class AggregateTagStorageTest {
+
+    private val storedTags = HashMap<ResourceId, Tags>()
```

Comment:
This one should be moved to `@Before fun beforeEach()` too, otherwise tests might interfere with each other through it.

Reply:
Yeah, this one is being reset at `@Before fun afterEach()` instead (line 77). But I completely understand the point, I'll move this map clearance to `beforeEach()` so it might get easier to find

Reply:
Oh, I didn't notice that. Now it's clearer anyway, thanks.